Validation/CaloTowers/test/run_onRelVal_data_cfg.py

```diff
@@ -0,0 +1,152 @@
+import os
+import FWCore.ParameterSet.Config as cms
+
+process = cms.Process("hcalval")
+process.load("Configuration.StandardSequences.Reconstruction_cff")
+process.load("Configuration.StandardSequences.Geometry_cff")
```

Comment:
would be great just to update the current branch - then it won't be forgotten (and that update will automatically enter this PR). Thanks.